.github/workflows/release.canary.yml

```diff
@@ -0,0 +1,27 @@
+name: Release canary and commit tags
+on:
+  push:
+    branches:
+      - main
+jobs:
+  docker-push-main:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Login to Docker Hub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKER_USER }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+      - name: Build and push canary tag
```

Comment:
Maybe, We can setup buildx to build faster. https://github.com/docker/setup-buildx-action

Reply:
It was mentioned as optional and recommended for multi platform builds so I skipped it, will add back.